src/style.css

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf"),
+    url("./fonts/Avenir-Heavy.ttf");
+}
+
+html {
+  --big-input-height: 70px;
+  --small-input-height: 42px;
+  --big-placeholder-padding: 62px;
+  --small-placeholder-padding: 33px;
+  --big-input-fz: 16px;
+  --small-input-fz: 14px;
+  --big-btn-heiht: 19px;
+  --big-btn-width: 19px;
+  --small-btn-height: 12px;
+  --small-btn-width: 11px;
```

Comment:
CSS variables are usually set in the `:root` block at the beginning of the file.